localstack/services/apigateway/apigateway_listener.py

```diff
@@ -121,6 +126,14 @@ def extract_path_params(path, extracted_path):
     return path_params
 
 
+def extract_query_string_params(path):
+    parsed_path = urlparse(path)
+    path = parsed_path.path
+    query_string_params = parse_qs(parsed_path.query)
```

Comment:
query_string_params is a dict of key(string), val(list).. it should be {string: string} unless there are []val intentionally?